packages/components/src/Tabs/src/Tab.tsx

```diff
@@ -0,0 +1,92 @@
+import { IconContext } from "@hopper-ui/icons";
+import { type StyledComponentProps, useStyledSystem } from "@hopper-ui/styled-system";
+import clsx from "clsx";
+import { type CSSProperties, type ForwardedRef, forwardRef, useContext, useId } from "react";
+import { type TabProps as RACTabProps, Provider, Tab as RACTab, useContextProps } from "react-aria-components";
+
+import { BadgeContext } from "../../Badge/index.ts";
+import { TagContext } from "../../tag/index.ts";
+import { Text, TextContext } from "../../typography/Text/index.ts";
+import { type BaseComponentDOMProps, cssModule } from "../../utils/index.ts";
+
+import { InternalTabsContext, TabContext } from "./TabsContext.ts";
+
+import styles from "./Tab.module.css";
+
+export const GlobalTabCssSelector = "hop-Tab";
+
+export interface TabProps extends
+    Omit<RACTabProps, "id" | "children" | "className" | "style">,
+    StyledComponentProps<BaseComponentDOMProps> {}
+
+function Tab(props: TabProps, ref: ForwardedRef<HTMLDivElement>) {
+    [props, ref] = useContextProps(props, ref, TabContext);
+    const { stylingProps, ...ownProps } = useStyledSystem(props);
+    const { variant, size, fluid } = useContext(InternalTabsContext) ?? {};
+    const contentId = useId();
```

Comment:
We don't need this. In s2, they need it because they have  collapsable tabs, which we don't.